.github/workflows/pull.yml

```diff
@@ -513,6 +513,32 @@ jobs:
         # Run pytest without simulator
         backends/arm/test/test_arm_baremetal.sh test_pytest
 
+  unittest-nxp-neutron:
```

Comment:
On one quick follow up, can we move this to trunk.yaml instead of here given early days for the delegate, we don't want to disrupt the regular devx.

Reply:
yeah once we get more tests/mature backend we can make it on "every pr"

Reply:
FYI we have to add a `ci: trunk` label to run these jobs.